conductr_cli/test/test_validation.py

```diff
@@ -64,7 +64,7 @@ def raise_hostname_lookup_error():
                                                    |Error: 127.0.0.1   localhost MagicBox
                                                    |Error: ::1         localhost MagicBox
                                                    |"""))
-        self.assertEqual(self.output(stderr), expected_output)
+        self.assertTrue(len(self.output(stderr)) > 0)
```

Comment:
I think we should continue asserting the message, especially since there is part of the message where the `socket.gethostname` is being inserted in the message.